### ISSUE TITLE
Support for eslint@^7.x.x

### ISSUE THREAD

**Issue by JustFly1984** [cvazac/eslint-plugin-react-perf#37]
@cvazac 
In yarn install logs in ci/cd I see the warning:

```sh
warning " > eslint-plugin-react-perf@3.2.3" has incorrect peer dependency "eslint@^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0".
```

<img width="559" alt="Screenshot 2020-07-13 16 00 01" src="https://user-images.githubusercontent.com/2255993/87285637-eed8ce00-c521-11ea-9b3f-a607da41254d.png">

We have eslint 7 installed and most of other plugins already support eslint 7

**Comment by JustFly1984** [comment]
please update eslint version. Is this package maintained?

**Comment by cvazac** [comment]
Thanks for the request - I'll make every effort to get to this this week.

**Comment by cvazac** [comment]
fixed in eslint-plugin-react-perf@3.2.4

**Comment by JustFly1984** [comment]
Thank you!
